tests/visitors/test_base.py

```diff
@@ -200,3 +204,28 @@ def test_import_with_alias(self) -> None:
             result = aliased_func()
         """
         self.assertCodemod(before, after)
+
+
+class OtherModuleRenameTransformer(BaseSimpleModuleRenameTransformer):
+    """Simple transformer renaming function from same module."""
+
+    rename_from = "django.dummy.module"
+    rename_to = "django.dummy.other_module"
+
+
+class TestSimpleModuleRenameTransformer(BaseVisitorTest):
```

Comment:
I'm not sure about the value of this 🤔 If I had to choose, I'd rather have a test for the `URLResolversTransformer` than this one. No much harm in keeping both though, let's leave it for now.

Reply:
there are many many more variations of module imports which need test cases.

As noted in https://github.com/browniebroke/django-codemod/issues/63 , there are very obvious gaps even in the handling of function imports.

And there are additional oddities with module imports (c.f. pyflakes test suite for imports which I mostly built)